newjersey/navigator.business.nj.gov
# feat: [#184858078] Nav Bar Updates Ethan Mobile

## Description

We wanted Ethans to have mobile nav bar with hamburger on the right side we corresponding animations (taken care of in a different commit already)

But now also truncated text with very long business names 

Also the business name being displayed on the mobile nav bar when not on the dashboard page. 



### Ticket



[Replace with ticket_id](https://www.pivotaltracker.com/story/show/184858078)

### Approach



### Steps to Test

Register as a user, make a really long business name in profile, see that it truncates on desktop and mobile

See that on mobile, the hamburger opens from the right side and the hamburger starts on the right side

Also see that when on non-dashboard pages on mobile the business name is displayed on the left side of the nav menu opposite the hamburger menu



### Notes



## Code author checklist
